src/stdlib.rs

```diff
@@ -46,6 +47,6 @@ impl<F: Field + From<u64>> IsZero<F> {
 
 impl<F: Field + Eq + Hash> IsZero<F> {
     pub fn wg<WGC>(&self, ctx: &mut StepInstance<F>, value: F) {
-        ctx.assign(self.value_inv, value.invert().unwrap_or(F::ZERO));
+        ctx.assign(self.value_inv, value.mi());
```

Comment:
The only `mi` implementation is for `T` in `halo2.rs`, but I don't see T used anywhere. Am I missing something?

Reply:
No, the reason is that our core code should not depend on halo2, so it is compatible with other backends.